core/tests/protractor_utils/LibraryPage.js

```diff
@@ -191,15 +191,15 @@ var LibraryPage = function() {
   };
 
   this.clickCreateActivity = async function() {
-    await createActivityButton.click();
+    await action.click('create Activity Button', createActivityButton);
     await waitFor.pageToFullyLoad();
   };
 
   this.clickExplorationObjective = async function() {
     await waitFor.elementToBeClickable(
       explorationObjective,
       'Exploration Objective takes too long to be clickable');
```

Comment:
This can be removed because action.click does this internally.

Reply:
alright will implement that